modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -1194,23 +1194,25 @@ private void _validateGlobalJSScriptElementAttributes(
 
 			if (Objects.equals(entry.getKey(), "src")) {
 				throw new GradleException(
-					"Use the 'url' property instead of the script attribute " +
-						"'src'");
+					"The key 'src' is not allowed as a script element " +
+						"attribute. Use 'url' instead.");
```

Comment:
Hey @drewbrokke I think this new error will confuse the user. It seems to indicate that instead of using a script element attributes called `src` they should use a script element attribute called `url`. This is not the case though. The shape of this config is like this:
```
liferay-sample-global-js:
    name: Liferay Sample Global JS
    type: globalJS
    url: example.js
    scriptElementAttributes:
        defer: true
        fetchPriority: auto
```
So, could we say something like: "The key 'src' is not allowed as a script element attribute. Use the 'url' property instead."?

cc: @thiago-buarqque

Reply:
I see, my mistake @ethib137. Is the `url` property required? If so then we may not need to mention it anyways. We could just say that `src` is not allowed. This particular block is just to validate the `scriptElementAttributes` values anyways.

Reply:
@ethib137 @thiago-buarqque Please see my follow-up PR [here](https://github.com/liferay-devtools/liferay-portal/pull/42) and let me know if that seems ok to you. Thanks!

Reply:
@drewbrokke 

> I see, my mistake @ethib137. Is the url property required?

Yes. 

> If so then we may not need to mention it anyways. We could just say that src is not allowed.

Mentioning to the dev that they should use the 'url' property instead of the 'src' attribute is clearer and we do not lose anything mentioning that.

```
The key 'src' is not allowed as a script element attribute. Use the 'url' property instead.
```

Please, fix the English in case of any mistakes

Reply:
My concern is that mentioning the `'url' property` may still be confusing, since it is not clear whether it is meant to use the top-level `url` property, or to add a `url` key to the `scriptElementAttribute` map. I can see a dev accidentally doing that and then being confused that it does nothing. A dev who is is using the `globalJS` type likely understands the role of the `url` property anyways, right?

Reply:
hmmm I see your point. What do you think @ethib137

Reply:
Yeah, that sounds okay to me.